faculty/clients/experiment.py

```diff
@@ -241,6 +241,24 @@ def list(self, project_id):
         endpoint = "/project/{}/experiment".format(project_id)
         return self._get(endpoint, ExperimentSchema(many=True))
 
+    def delete(self, project_id, experiment_id):
+        """Delete a specified experiment.
+
+        Parameters
+        ----------
+
+        project_id : uuid.UUID
+        experiment_id : int
+
+        Returns
```

Comment:
Typically, if a method doesn't return anything (== returns None), we'd just omit the 'Returns' block.

Reply:
okay